[Terra-clinical-detailview] Fixes issue of empty space when no elements provided for title section (#892)
cerner/terra-clinical: packages/terra-clinical-detail-view/CHANGELOG.md

## Unreleased

* Fixed

>   * Fixed issue of empty space when no elements specified for title section.

Reviewer: ```suggestion
  * Fixed extra empty space showing when no elements are specified for title section.
```

Author: updated : https://github.com/cerner/terra-clinical/commit/c598ecc9301aad24ee248a1a5a560192d371c448#diff-597dea7fda801a6975eff274534403ab62eb826af80dcae25cd5ae7c4f6cde98
